executor/common.h

```diff
@@ -647,9 +650,9 @@ static void loop(void)
 // clang-format off
```

Comment:
Is this part still relevant? Or we can remove clang-format annotations?

Reply:
Removed.